### ISSUE TITLE
make install fails with error: No such file or directory

### ISSUE THREAD

**Issue by N4tr0n** [chrisbarrett/swift-mode#120]
Hi,

I was trying to install swift-mode using the instructions for manual installation and the `make install` step fails with the following error:
```
Opening input file: No such file or directory, /Users/nathan/Projects/swift-mode/dist/swift-mode-0.4.0snapshot.tar
make: *** [install] Error 255
```
The `make` step is successful. According to [the Cask documentation](http://cask.readthedocs.org/en/latest/guide/usage.html), `cask package` will only create a TAR archive if the package contains more than one file otherwise it just copies the emacs lisp file to the dist directory. Simply changing the file extension in the `Makefile` clears up the error. I could make a pull request but I wasn't sure if I was missing something. I'm using Cask 0.7.4 from Homebrew and the latest swift-mode sources on OS X El Capitan.

Thanks,
Nathan

**Comment by ap4y** [comment]
Yep, tried myself, looks like a bug. I'll appreaciate if you submit a PR. Thanks!

**Comment by ap4y** [comment]
fixed in #122
